docs/usage/dataclasses.md

```diff
@@ -317,10 +329,10 @@ print(repr(m))
 #> Model(dc=DC(a=ArbitraryType(value=3), b='qwe'), other='other')
 ```
 
-## Initialize hooks
+## Initialization hooks
```

Comment:
Are these `hooks` which are initialized? _If yes_, then `Initialize hooks` is the correct title.

Are these `hooks which run on class initialization`? _If yes_, `Initialization hooks` is the correct title.

Reply:
The latter in this case